cmd/sku/put.go

```diff
@@ -0,0 +1,91 @@
+package sku
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/spf13/cobra"
+
+	"github.com/devigned/pub/cmd/args"
+	"github.com/devigned/pub/pkg/partner"
+	"github.com/devigned/pub/pkg/service"
+	"github.com/devigned/pub/pkg/xcobra"
+)
+
+type (
+	putPlanArgs struct {
+		Publisher   string
+		Offer       string
+		SkuFilePath string
+	}
+)
+
+func newPutCommand(sl service.CommandServicer) (*cobra.Command, error) {
+	var oArgs putPlanArgs
+	cmd := &cobra.Command{
+		Use:   "put",
+		Short: "create a SKU",
+		Run: xcobra.RunWithCtx(func(ctx context.Context, cmd *cobra.Command, args []string) error {
+			bits, err := ioutil.ReadFile(oArgs.SkuFilePath)
+			if err != nil {
+				sl.GetPrinter().ErrPrintf("%v\n", err)
+				return err
+			}
+
+			var plan partner.Plan
+			if err := json.Unmarshal(bits, &plan); err != nil {
+				sl.GetPrinter().ErrPrintf("unable to unmarshal JSON plan into partner.Plan: %v", err)
```

Comment:
"Plan" might be weird to a user. Perhaps, this should be SKU since that is the way the CLI talks about this type.

Reply:
Is this better?
"unable to unmarshal JSON from sku file into an object: %v", err
I don't want to mislead by saying we're trying to unmarshall into an object with SKU in the name.